flink-tests/src/test/java/org/apache/flink/test/checkpointing/ChangelogLocalRecoveryITCase.java

```diff
@@ -142,7 +142,6 @@ public void testRestartTM() throws Exception {
         // wait job for doing materialization.
         waitUntilCondition(
                 () -> !getAllStateHandleId(firstJobGraph.getJobID(), miniCluster).isEmpty());
-        miniCluster.triggerCheckpoint(firstJobGraph.getJobID()).get();
```

Comment:
If we just remove this, IIUC, the case will always just test local recovery about the materialization part without non-materialization part ?
Could we produce a checkpoint including two parts ? e.g. Using ControlledSource in ChangelogRecoveryITCaseBase. Or other ways ?

Reply:
For this test, the checkpoint interval is 200ms and the materialization interval is 800ms. So, at least one non-materialization part has been produced before we get the materialization part.